.travis.yml

```diff
@@ -1,10 +1,15 @@
-before_install:
-   - sudo apt-get update -qq
-   - sudo apt-get install -qq uuid-dev
+sudo: false
+branches:
+    only:
+      - master
```

Comment:
Travis will stuff run PRs.  What this prevents is travis running two different jobs (a PR job and a "push" job) every time you update the branch that your job is running on.  Since our entire account has a limited number of concurrent jobs this helps.